assets/queries/cloudFormation/CMK_Unencrypted_Storage/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "CMK_Unencrypted_Storage",
+  "queryName": "CMK Unencrypted Storage",
+  "severity": "HIGH",
+  "category": "Encryption and Key Management",
+  "descriptionText": "Ensure that storage is encrypted by KMS on instances that, based on their name, might host a database. ",
+  "descriptionUrl": "#"
```

Comment:
Missing docs link

Reply:
Yeah, I am not sure what link to put here. since it is catching results in many types of database resources. Do you have a suggestion??